src/components/PotentialCareers/PotentialCareers.scss

```diff
@@ -0,0 +1,39 @@
+@use "../../styles/_typography" as *;
+@use "../../styles/_variables.scss" as *;
+@use "../../styles/_reset" as *;
+
+.career {
+  padding: 1rem;
+  width: 23.063rem;
+  height: fit-content;
+
+  background: #ffffff;
+
+  box-shadow: 0px 20px 40px rgba(208, 209, 216, 0.24);
+  border-radius: 10px;
```

Comment:
Can you change those lines to rem as well

Reply:
Made the changes